store/src/java-test/com/zimbra/cs/filter/RuleManagerAdminFilterTest.java

```diff
@@ -186,22 +185,17 @@ public void invalidRequireComand() throws Exception {
     String[] variableScripts = {
             // admin-before
               "require [\"tag\", \"log\", \"variables\"];"
-            + "set \"beforevar\" \"foo\";"
-            + "tag \"admin-before1-${beforevar}\";",
+            + "set \"var\" \"foo\";"
```

Comment:
Mmm... Still I couldn't see your new fix (f10b1cfe969285030fe895f4dd54e49ebc656cde) recovered the zcs-1378's change.  

It seems https://github.com/Zimbra/zm-mailbox/commit/f10b1cf just added one blank line.